iosApp/SurveyTests/Sources/Specs/Presentation/Modules/SurveySelection/SurveySelectionViewDataSourceSpec.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  SurveySelectionViewDataSourceSpec.swift
+//  Survey
+//
+//  Created by Bliss on 15/12/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Nimble
+import Quick
+import Shared
+
+@testable import Survey
+
+final class SurveySelectionViewDataSourceSpec: QuickSpec {
+
+    override func spec() {
+
+        var getCurrentDateUseCase: GetCurrentDateUseCaseKMMMock!
+        var getProfileUseCase: GetProfileUseCaseKMMMock!
+        var surveySelectionViewModel: SurveySelectionViewModel!
+        var dataSource: SurveySelectionView.DataSource!
+
+        describe("a Survey Selection View Data Source") {
+
+            beforeEach {
+                getCurrentDateUseCase = GetCurrentDateUseCaseKMMMock()
+                getProfileUseCase = GetProfileUseCaseKMMMock()
+                surveySelectionViewModel = SurveySelectionViewModel(
+                    getCurrentDateUseCase: getCurrentDateUseCase,
+                    getProfileUseCase: getProfileUseCase,
+                    dateTimeFormatter: DateTimeFormatterImpl()
+                )
+                dataSource = .init(
+                    viewModel: surveySelectionViewModel
+                )
+            }
+
+            describe("its init") {
```

Comment:
Shall we apply the agreed [convention](https://github.com/nimblehq/compass/discussions/1048) for describe function? 🤔 And maybe the rest of places too? 🙏
```suggestion
            describe("its init() call") {
```

Reply:
I don't think we agreed on that yet, with () it doesn't look good 😆, without () it looks more similar to Android side.